cogboard-webapp/src/components/DropdownField.js

```diff
@@ -12,7 +12,8 @@ const DropdownField = props => {
     name,
     children,
     dropdownItems,
-    itemsUrl
+    itemsUrl,
+    ...other
```

Comment:
Is this change should be a part of this commit?

Reply:
Yes - 

> and all the other props given